test/channel_test.exs

```diff
@@ -14,4 +14,9 @@ defmodule ChannelTest do
     assert {:ok, chan} = Channel.open(meta[:conn])
     assert :ok = Channel.close(chan)
   end
+
+  test "open channel for direct consumer", meta do
```

Comment:
I think we can delete this test as it's covered by DirectConsumerTest. Or please use simple_receiver if you want to keep the test.

Reply:
Done :)